### PR TITLE
Setup the new Send Money page and header titles

### PR DESCRIPTION
**Pull request by nickmurray47**
### Details
Adds the Send Money Page container and updates the header title steps to reflect the new flow

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/169170
$ https://github.com/Expensify/Expensify/issues/169174

### Tests
1. Test the new route by going to `/iou/send/{currentChatReportID}`, e.g. `/iou/send/182`. Verify this opens up a new IOU Send modal.
2. Go through the flow of creating a new IOU
2. Verify the correct header titles appear, "Send Money" and "Send $amount". *the button text will be updated from "Request" to "Send" in a future PR.
Screen Shot 2021-07-01 at 4 57 15 PM
Screen Shot 2021-07-01 at 5 09 22 PM

### QA Steps
Same as above. Will also test the full flow once the payment and DM/Global Create menu's are hooked up.

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

### CODE REVIEW

File: src/pages/iou/IOUModal.js [hunk 1]
```diff
@@ -159,18 +163,27 @@ class IOUModal extends Component {
     getTitleForStep() {
         const currentStepIndex = this.state.currentStepIndex;
         if (currentStepIndex === 1 || currentStepIndex === 2) {
+            const formattedAmount = this.props.numberFormat(
+                this.state.amount, {
+                    style: 'currency',
+                    currency: this.state.selectedCurrency.currencyCode,
+                },
+            );
+            if (this.props.iouType === 'send') {
```
Comment (MariaHCD) [inline src/pages/iou/IOUModal.js:172]: NAB: Should we have the IOU type `send` in a constant?

**Review by MariaHCD — APPROVED** [review]
LGTM! Testing works as expected.

File: src/pages/iou/IOUModal.js (hunk 1 above)
Comment (nickmurray47) [reply src/pages/iou/IOUModal.js:172]: Good point! I have another PR dependent on this one, so will make that change there.

**Comment by roryabraham** [comment]
Unfortunately the deploy comments are not working right now. This was deployed to staging yesterday.

File: src/languages/en.js [hunk 2]
```diff
@@ -136,6 +136,7 @@ export default {
         confirm: 'Confirm',
         splitBill: 'Split Bill',
         requestMoney: 'Request Money',
+        sendMoney: 'Send Money',
```
Comment (Beamanator) [inline src/languages/en.js:139]: Creepin' on your PR :D Did you mean to also add a translation in `es.js`?

Comment (nickmurray47) [reply src/languages/en.js:139]: Was planning on adding all the translations in one big translation PR at the end!

Comment (Beamanator) [reply src/languages/en.js:139]: Sweet!

**Pull request merged**